pkg/backup/request.go

```diff
@@ -20,6 +20,7 @@ import (
 	"fmt"
 	"sort"
 
+	veleroapiv1 "github.com/vmware-tanzu/velero/api/v1"
```

Comment:
if we do go ahead with two separate packages for the API types, we need to come up with a clearer way to alias them :)

Reply:
Yes, I'm very open to have a better alias name. For lack of a great idea, I turned `velero/api/v1` into `veleroapiv1`, which (obvs) follows the order of the directories (which doesn't mean is great but has a meaning), and also was NOT the `velerov1api` which we use for our existing api package.